lib/renderer/init.js

```diff
@@ -123,7 +123,8 @@ if (nodeIntegration === 'true') {
     delete global.process
     delete global.setImmediate
     delete global.clearImmediate
-    return delete global.global
+    delete global.Buffer
```

Comment:
Simply deleting it from `global` is not enough, this breaks code in preload script that uses `Buffer` after the web page has loaded. We also need to add `Buffer` to the list in https://github.com/electron/node/commit/5b76ffffd18a5359062b76e764b8ffdab1904857.